dll/win32/shell32/folders/CFSFolder.cpp

```diff
@@ -1066,9 +1066,12 @@ HRESULT WINAPI CFSFolder::CompareIDs(LPARAM lParam,
                 result = pData1->u.file.uFileTime - pData2->u.file.uFileTime;
             break;
         case SHFSF_COL_FATTS:
-            return SHELL32_CompareDetails(this, lParam, pidl1, pidl2);
+            result = SHELL32_CompareDetails(this, lParam, pidl1, pidl2);
+            if (result == 0)
+                result = pData1->u.file.uFileAttribs - pData1->u.file.uFileAttribs; // Attributes without a "UI letter"
```

Comment:
What do you mean by "UI letter"? the "R/A/S/H/..." of the read-only/archive/... attributes?

Reply:
Yes, correct.